taskfiles/deps.yaml

```
@@ -0,0 +1,151 @@
+version: "3"
+
+tasks:
+  install-all:
+    desc: "Install all dependencies required by clp-ffi-js."
+    run: "once"
+    cmds:
+      - "rm -rf '{{.G_DEPS_CMAKE_SETTINGS_DIR}}'"
+      - "mkdir -p '{{.G_DEPS_CMAKE_SETTINGS_DIR}}'"
+      - task: ":utils:cmake:install-deps-and-generate-settings"
+        vars:
+          CMAKE_SETTINGS_DIR: "{{.G_DEPS_CMAKE_SETTINGS_DIR}}"
+          CMAKE_SETTINGS_FILE: "{{.G_DEPS_CMAKE_SETTINGS_FILE}}"
+          DEP_TASK: "deps:install-all-parallel"
+
+  install-all-parallel:
+    internal: true
+    deps:
+      - "download-boost"
+      - "download-clp"
+      - "download-fmt"
+      - "download-nlohmann-json"
+      - "download-spdlog"
+      - "download-ystdlib"
+      - "download-zstd"
+
+  download-boost:
+    internal: true
+    vars:
+      LIB_NAME: "boost"
+      BOOST_OUTPUT_DIR: "{{.G_DEPS_DIR}}/{{.LIB_NAME}}-src"

```

```suggestion
      BOOST_OUTPUT_DIR: "{{.G_DEPS_DIR}}/{{.LIB_NAME}}-src"
      BOOST_VERSION: "1.88.0"
```

* Since we're using it in two places.
* Chose 1.88 since that's the latest.